src/pykoala/plotting/plot_plot.py

```diff
@@ -0,0 +1,418 @@
+    chlines : list of strings (default = [])
+        Color of the horizontal lines
+    cvlines : list of strings (default = [])
+        Color of the vertical lines
+    axvspan : list of floats (default = [[0,0]])
+        Shades the region between the x positions specified
+    caxvspan : list of strings (default = [])
+        Color of the shaded regions     
+    hwidth: float (default =1)
+        thickness of horizontal lines
+    vwidth: float (default =1)
+        thickness of vertical lines
+    frameon : boolean (default = False)
+        Display the frame of the legend section
+    loc : string or pair of floats or integer (default = 0)
+        Location of the legend in pixels. See matplotlib.pyplot.legend() documentation
+    ncol : integer (default = 6)
+        Number of columns of the legend
+    label : string or list of strings (default = "")
+        Specify labels for the graphs
+    title_fontsize: float (default=12)
+        Size of the font of the title
+    label_axes_fontsize: float (default=10)
+        Size of the font of the label of the axes (e.g. Wavelength or Flux)
+    axes_fontsize: float (default=10)
+        Size of the font of the axes (e.g. 5000, 6000, ....)
+    tick_size: list of 4 floats (default=[5,1,2,1])
+        [length_major_tick_axes, thickness_major_tick_axes, length_minor_tick_axes, thickness_minor_tick_axes]   
+        For defining the length and the thickness of both the major and minor ticks
+    axes_thickness: float (default=0)
+        Thickness of the axes
+    save_file : string (default = none)
+        Specify path and filename to save the plot
+    path: string  (default = "")
+        path of the file to be saved
+    fig_size : float (default = 12)
+        Size of the figure
+    warnings : boolean (default = True)
+        Print the warnings in the console if something works incorrectly or might require attention 
+    show : boolean (default = True)
+        Show the plot
+    statistics : boolean (default = False)
+        Print statistics of the data in the console
+       
+   """
+      
+   if fig_size == "big":
+       fig_size=20
+       label_axes_fontsize=20 
+       axes_fontsize=15
+       title_fontsize=22
+       tick_size=[10,1,5,1]
+       axes_thickness =3
+       hwidth =2
+       vwidth =2
+
+   if fig_size in ["very_big", "verybig", "vbig"]:
+       fig_size=35
+       label_axes_fontsize=30 
+       axes_fontsize=25
+       title_fontsize=28 
+       tick_size=[15,2,8,2]
+       axes_thickness =3
+       hwidth =4 
+       vwidth =4
+
+   if fig_size != 0 : 
+       if type(fig_size) == list:
+           plt.figure(figsize=(fig_size[0], fig_size[1]))
+       else:     
+           plt.figure(figsize=(fig_size, fig_size/2.5))
+   
+   if np.isscalar(x[0]) :
+       xx=[]
+       for i in range(len(y)):
+           xx.append(x)
+   else:
+       xx=x
+
+   if xmin is None : xmin = np.nanmin(xx[0])
+   if xmax is None : xmax = np.nanmax(xx[0])  
+   
```

Comment:
`xmin = np.nanmin(wavelent_list)`
Why only the first wavelength vector in the list?